locales/en-US/snippet.properties

```diff
@@ -26,7 +26,7 @@ snippet_var_e=Mozilla, the organization behind Firefox, is proudly non-profit, n
 
 # Variant F “Hat Tip Long”:
 # example for "{currency}{amount}" in English (US) is "$3". Context for "tip our hat" is related to the dapper fox's hat. It's an idiomatic English expression used to admire and respect what someone has done. Link to Dapper fox on one of our staging http://scottdowne.github.io/eoy-snippets/2015-spectacular-snippet-2/
-snippet_var_f=HELLO there: We know you love Firefox, but did you know a non-profit called Mozilla built Firefox? Thanks to donations, Mozilla does a lot more than build this awesome browser. We teach people to code, fight for online privacy, and protect this amazing thing called the Web for future generations. Just a few times a year Mozilla asks for donations, and usually only a tiny portion of Firefox users give. <highlight>If everyone reading this chipped in {currency}{amount} to help Mozilla, we could wrap this fundraiser up in under an hour and tip our hat to you! <i>Thank you</i>.</highlight>
+snippet_var_f=HELLO there: We know you love Firefox, but did you know a non-profit called Mozilla built Firefox? Thanks to donations, Mozilla does a lot more than build this awesome browser. We teach people to code, fight for online privacy, and protect this amazing thing called the Web for future generations. Just a few times a year Mozilla asks for donations, and usually only a tiny portion of Firefox users give. <em>If everyone reading this chipped in {currency}{amount} to help Mozilla, we could wrap this fundraiser up in under an hour and tip our hat to you!</em> <i>Thank you</i>.
```

Comment:
We could probably leave `<highlight>` as such, but I don't want to have to constantly explain and change it to everyone on the snippet team whenever we try to use it. One source of truth please.